Makefile

```diff
@@ -46,6 +56,25 @@ $(VERSION).tar.$(SRC_EXT):
 $(subst rpm,%,$(RPMS)): $(SPEC) $(SOURCES)
 	rpmbuild -bb $(COMMON_RPM_ARGS) $(RPM_BUILD_OPTIONS) $(SPEC)
 
+$(subst deb,%,$(DEBS)): $(SPEC) $(SOURCES)
+	# Unpack tarball
+	export TAR_OPTIONS="--owner=0 --group=0 --numeric-owner"
+	mkdir -p ${DEB_BUILD}
+	tar -C $(DEB_BUILD) --strip-components=1 --atime-preserve \
+	  -xpf _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+	# patch tarball
+	for f in _topdir/SOURCES/*.patch; do \
+	  patch -d $(DEB_BUILD) -p1 < $$f; \
+	  done
+	tar -C _topdir/BUILD \
+	  -czf _topdir/BUILD/$(NAME)-$(VERSION).tar.$(SRC_EXT) \
+	  $(NAME)-$(VERSION)
```

Comment:
This unpack-patch-pack operation seems like even if it does create a `debian/patches/` subdir with a patch in it, it's going to glom them all into a single patch.  In the case of there being several patches, it would be nicer to keep them separate in `debian/patches/`.

Is there any option to this `debmake --binaryspec` to give it the patches separately for it to put into `debian/patches/`?